packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -190,6 +189,29 @@ export const Stackbox = () => {
     tokenListWithBalances,
   ]);
 
+  /**
+   * startDate and endDate handler when the user
+   * is idle for 1 minute or more
+   */
+  useEffect(() => {
+    function oneMinuteInterval() {
+      const startDateInThePast = startDateTime.getTime() <= Date.now();
+      if (startDateInThePast) {
+        setStartDateTime(getDatePlusMinsOffset(startDateTime, ONE_MINUTE));
+        setEndDateTime(getDatePlusMinsOffset(endDateTime, ONE_MINUTE));
+      }
+    }
+
+    oneMinuteInterval();
+
+    const id = setInterval(oneMinuteInterval, ONE_MINUTE_IN_MS);
+
+    return () => {
+      clearInterval(id);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Not sure if we want this. Was editing the date and it auto updated while I was editing it.

I think we should update the start time only, if we want to keep it in the present. Also I would move it to the present, and not add 1 minute.

Reply:
I think we don't need to update the UI if in the past, we just handle when submit
- if start time is on the past set it to the current time.
The confirm model will show the starting time and will allow verification

IMO this makes the code more complex without giving much upside